### ISSUE TITLE
Configurable email recipient groups

### ISSUE THREAD

**Issue by githubert** [githubert/lettersnail#6]
Currently it is quite a hassle to manage messages to multiple recipients which may later need some additions or changes.
It should be possible to support a special notation such as `group:foo` which will expand to a list of addresses when the message is sent out. The special notation should be supported for all fields that accept multiple email addresses, (which excludes `from`. and probably `reply-to`).

It should be possible to specify these group-addresses along side with normal addresses, for example, `to: baz@example.com, group:foo, foo@example.com`.

This will probably be one of the features where the configuration resides completely in the INI-file. There should be some notation for the INI-file such as `group.foo = bar@example.com. foo@example.com`.
Recursion (`group.foo = group:bar, group:baz, foo@example.com`) is probably not warranted here.

To recap, **steps to be done**:
- [ ] INI-file: Determine best way to represent this.
- [ ] INI-file: Display an error if recursion is attempted.
- [ ] Implement `group:` notation for `to`, `cc` and `bcc`.
- [ ] Replace groups with actual addresses in debug and logging output.
